synapse/http/matrixfederationclient.py

```diff
@@ -649,12 +651,12 @@ async def _send_request(
 
                     if retries_left and not timeout:
                         if long_retries:
-                            delay = 4 ** (MAX_LONG_RETRIES + 1 - retries_left)
-                            delay = min(delay, 60)
+                            delay = 4 ** (self.max_long_retries + 1 - retries_left)
+                            delay = min(delay, self.max_long_retry_delay)
                             delay *= random.uniform(0.8, 1.4)
                         else:
                             delay = 0.5 * 2 ** (MAX_SHORT_RETRIES - retries_left)
```

Comment:
Since we're here can we also pull out `MAX_SHORT_RETRIES` please? Mostly for symmetry as we can configure it for the long retry scheme